anni-playback/src/sources/cached_http/cache.rs

```diff
@@ -84,6 +90,65 @@ impl CacheStore {
             Err(io::Error::new(ErrorKind::Other, "invalid cache"))
         }
     }
+
+    pub fn store_info(&self, track: RawTrackIdentifier, kind: &str, value: &str) -> io::Result<()> {
+        let path = {
+            let mut p = self.loaction_of(track.copied());
+            p.set_extension("info");
+            p
+        };
+
+        let mut info = match File::open(&path) {
+            Ok(f) => read_info(&f)?,
+            Err(e) if e.kind() == ErrorKind::NotFound => HashMap::with_capacity(1),
+            Err(e) => return Err(e),
+        };
+        info.insert(kind.to_owned(), value.to_owned());
+
+        let mut writer = File::options()
+            .write(true)
+            .truncate(true)
+            .create(true)
+            .open(path)?;
+
+        for (kind, value) in info {
+            log::debug!("{track}: stored {kind} = {value}");
+            writeln!(writer, "{kind}:{value}")?;
+        }
+
+        Ok(())
+    }
+
+    pub fn acquire_info(&self, track: RawTrackIdentifier) -> io::Result<HashMap<String, String>> {
+        let path = {
+            let mut p = self.loaction_of(track);
+            p.set_extension("info");
+            p
+        };
+
+        match File::open(&path) {
+            Ok(f) => read_info(&f),
+            Err(e) if e.kind() == ErrorKind::NotFound => Ok(HashMap::new()),
+            Err(e) => return Err(e),
+        }
+    }
+}
+
+fn read_info(f: &File) -> io::Result<HashMap<String, String>> {
+    let reader = BufReader::new(f);
+
+    let mut info = HashMap::new();
+
+    for line in reader.lines() {
```

Comment:
What about using a full-featured format, like JSON so that we can change it in the future 🤔

Reply:
Sounds good. Shall we store them in a schema like `{ "key1": value1, "key2": value2, ... }`?